### ISSUE TITLE
Modify diego-release to use modified auctioneer

### ISSUE THREAD

**Issue by ilackarms** [mesos/cloudfoundry-mesos#2]
Rather than manually replacing the auctioneer binary (noted here: https://github.com/mesos/cloudfoundry-mesos/blob/master/docs/getting-started.md), I would recommend forking and modifying the Diego BOSH release to deploy the modified auctioneer itself.
-You will need to modify the job template for the auctioneer https://github.com/cloudfoundry-incubator/diego-release/blob/develop/jobs/auctioneer/templates/auctioneer_ctl.erb
-The only other step you should need would be to replace the auctioneer submodule in the diego-release with your auctioneer (it would make sense to fork the CF auctioneer and put your changes in there to streamline this process).

**Comment by jianhuiz** [comment]
What we are considering is to create a separate release instead of modifying diego-release, and deploy this auctioneer job from that release. This is however not our priority right now, we welcome any pull request.

**Comment by Amit-PivotalLabs** [comment]
I agree with @jianhuiz, having a separate release for the auctioneer probably makes more sense.  If you fork the whole diego-release then you have to continually work to keep your fork up-to-date.
